packages/ddp-client/livedata_connection_tests.js

```diff
@@ -819,34 +819,48 @@ if (Meteor.isClient) {
 
     startAndConnect(test, stream);
 
-    var methodCallbackFired = false;
-    var methodCallbackErrored = false;
+    var firstMethodCallbackFired = false;
+    var firsMethodCallbackErrored = false;
```

Comment:
Looks like a typo here - `firsMethodCallbackErrored` should be `firstMethodCallbackErrored`.